build-scripts/ubuntu-1604/build-indy-node.sh

```diff
@@ -27,6 +27,7 @@ fpm --input-type "python" \
     --exclude "*.pyo" \
     --depends at \
     --depends iptables \
+    --depends libsodium18 \
```

Comment:
I think we can now remove libsodium18 from other places (like Vagrant) so that we test that it's installed properly.

Reply:
Yes. After this fix, libsodium18 installation steps in vagrant/docker will not be needed.